lib/diagnostic.md

```diff
@@ -54,7 +65,7 @@ Please write a method that traverses from the lister DOM node (one of the list
 item elements) to the body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('body').hide();
 <!-- your answer ends here -->
```

Comment:
I was looking for starting at $('.lister').parent().parent().hide() but your answer works too.